src/index.html

```diff
@@ -7,8 +7,77 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap">
+    <link rel="stylesheet" href="images/logo.png">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="Logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple" class="nav__link is-active apple">
+              Apple
+            </a>
```

Comment:
```suggestion
            <a
              href="#apple" 
              class="nav__link is-active apple"
            >
              Apple
            </a>
```
<img width="1185" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/2c20c8a6-b9dc-4e0c-a4e3-605f7b9bab52">